packages/next/src/build/webpack/config/blocks/css/loaders/global.ts

```diff
@@ -24,28 +24,42 @@ export function getGlobalCssLoader(
     )
   }
 
-  // Resolve CSS `@import`s and `url()`s
-  loaders.push({
-    loader: require.resolve('../../../../loaders/css-loader/src'),
-    options: {
-      postcss,
-      importLoaders: 1 + preProcessors.length,
-      // Next.js controls CSS Modules eligibility:
-      modules: false,
-      url: (url: string, resourcePath: string) =>
-        cssFileResolve(url, resourcePath, ctx.experimental.urlImports),
-      import: (url: string, _: any, resourcePath: string) =>
-        cssFileResolve(url, resourcePath, ctx.experimental.urlImports),
-    },
-  })
+  if (ctx.experimental.useLightningcss) {
```

Comment:
We should probably reason about postcss config existing as well, especially the behavior there, for example should it run both postcss and lightningcss in a two-pass or should it run one or the other.

Reply:
Is [`296bf83` (#61327)](https://github.com/vercel/next.js/pull/61327/commits/296bf83716472f0f5f0fad85ec1e9e369d6d6735) what you want?